server/api/src/main/java/io/smallrye/graphql/api/GraphQlErrorCode.java

```diff
@@ -0,0 +1,14 @@
+package io.smallrye.graphql.api;
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Retention(RUNTIME)
+@Target({ TYPE, ANNOTATION_TYPE })
+public @interface GraphQlErrorCode {
```

Comment:
Remove GraphQl (to be consistent with other annotations) So just `@ErrorCode`

Reply:
done